pkg/internal/utils/utils.go

```diff
@@ -29,10 +12,12 @@ const (
 	MDFormat      = "md"
 )
 
-// diff format common const
+// Ingress Controller const - the name and namespace of an ingress-controller pod
 const (
-	DotHeader  = "digraph {"
-	DotClosing = "}"
+	//  The actual ingress controller pod is usually unknown and not available in the input resources for the analysis.
+	// IngressPodName and IngressPodNamespace are used to represent that pod with those placeholder values for name and namespace.
+	IngressPodName      = "ingress-controller"
+	IngressPodNamespace = "ingress-controller-ns"
 )
```

Comment:
why cannot this (`IngressPodName`, `IngressPodNamespace`) remain in `pkg/netpol/common` ? It seems to be used only by `connlist` and `diff`.

Reply:
in the new command_test version (#258), we have test with input `focusWorkload: "ingress-controller"`, so I am thinking assign it with the `IngressPodName const`

Reply:
It is more related to packages under `pkg/netpol`, so I would keep it there. The api-based tests can use this const, and the cli test can just use this as hard-coded string.